tasks.py

```diff
@@ -155,6 +161,11 @@ def type_check(
         print("\n".join(DEFAULT_PACKAGES_TO_TYPE_CHECK))
         raise invoke.Exit(code=0)
 
+    if clear_cache:
+        print("  Clearing the mypy cache ... ", end="")
+        ctx.run(" ".join(["rm", "-rf", ".mypy_cache"]))
```

Comment:
Only issue with this is it isn't windows friendly.
Could update this to do this with pure python. With one of `shutil`/`os`/`pathlib`.

Reply:
I'm fine with this. If the goal is for contributors to use this then we should use one of your alternatives.